tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
Remove comments from this file pls

Reply:
i dont want, they can be usefull

Reply:
It's a mess of comments, not a config file